pkg/cli/admin/upgrade/channel/channel.go

```diff
@@ -0,0 +1,127 @@
+// Package channel contains a command for setting a cluster's update channel.
+package channel
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	configv1client "github.com/openshift/client-go/config/clientset/versioned"
+	"github.com/spf13/cobra"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	kcmdutil "k8s.io/kubectl/pkg/cmd/util"
+	"k8s.io/kubectl/pkg/util/templates"
+)
+
+func NewOptions(streams genericclioptions.IOStreams) *Options {
+	return &Options{
+		IOStreams: streams,
+	}
+}
+
+func New(f kcmdutil.Factory, streams genericclioptions.IOStreams) *cobra.Command {
+	o := NewOptions(streams)
+	cmd := &cobra.Command{
+		Use:   "channel CHANNEL",
+		Short: "Set or clear the update channel",
```

Comment:
```
oc adm upgrade channel candidate-4.6
```
Is not intuitive IMO. We need a verb or flag to make it intuitive  e.g. `oc adm upgrade channel  --to candidate-4.6`

Reply:
I was trying to compile this PR to test the user experience but getting in to some error because I am sing go 1.15.2

Reply:
Rebased onto master, and it compiles in the CI presubmits.  What error were you seeing?

/remove-lifecycle stale

Reply:
I can compile the PR successfully now.